.github/workflows/check-labels.yml

```
@@ -0,0 +1,31 @@
+
+jobs:
+  check-labels:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # Fetch all history for all branches
+
+      - name: Detect Changes
+        id: detect
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:5:30: Double quote to prevent globbing and word splitting

(shellcheck)